common/models/workspace.js

```diff
@@ -33,6 +33,15 @@ module.exports = function(Workspace) {
     var ViewDefinition = app.models.ViewDefinition;
     var TEMPLATE_DIR = path.join(__dirname, '..', '..', 'templates', 'projects');
     var DEFAULT_TEMPLATE = 'api-server';
+    var DEFAULT_LB_VERSION = '2.x';
+    var DEPENDENCY_3_X = {
+      'loopback': '^3.0.0-alpha.1',
+      'loopback-datasource-juggler': '^3.0.0-alpha.1',
```

Comment:
Uff, I don't know what's the full list of dependencies. Here are the differences between 2.x and 3.x:

```diff
- "loopback": "^2.22.0",
- "loopback-datasource-juggler": "^2.39.0",
+ "loopback": "^3.0.0-alpha.1",
```

See how `givenLB3Workspace` and `givenLB2Workspace` helpers work, they should be already configuring the right set of dependencies.